packages/es-components/babel.config.js

```diff
@@ -1,11 +1,39 @@
-const wtwConfig = {
-  extractFormatMessage: false,
-  transformFormatMessage: false
-};
+module.exports = function(api) {
+  const modules = api.env('test') ? 'auto' : false;
+
+  const presets = [
+    [
+      '@babel/preset-env',
+      {
+        useBuiltIns: 'usage',
+        corejs: 3,
+        modules: modules,
+        debug: false
+      }
+    ],
```

Comment:
Besides changing the usage of corejs from v2 to v3 in both `preset-env` and `transform-runtime`, how much of these configuration changes do we need and what can be elevated to `babel-preset-wtw-im`?

Reply:
Looking through the configs, I think I only see 2 other things that are different:
- I removed `targets` from the babel config in favor of using the `browserslist` setting in `package.json`. (https://babeljs.io/docs/en/babel-preset-env#browserslist-integration)
- I added `fileName: false` to the styled-components config, though I think I should remove that. It probably ought to be using either `displayName` OR `fileName`.

Reply:
I suppose using `targets` in our `babel-preset-wtw-im` allows us to set the default for everyone more easily than telling people how to use the browserslist config.

Reply:
Removing `targets` from `babel-preset-wtw-im` would be a breaking change in my eyes. While the API won't be changing, per se, but it's how we're telling downstream projects using our preset to compile with IE11 in mind. While a part of me is fine with updating the targets to use `> 0.5%, not dead` I am slightly concerned with accidentally pushing public builds that break on ie11 before we've had a chance to create a graceful unsupported browser page.

Reply:
So how should we approach this? Do we update wtw-im-babel to use core-js 3, then update es-components to use it? Do I drop this back down to core-js 2 and we worry about v3 later? Leave it as is for now?

Reply:
We can still override the babel preset in our config.

```js
['babel-preset-wtw-im', {
  env: {
    corejs: 3,
    modules
  }
}]
```

I think we can go forward with this as an intermediary before updating the preset to use core-js 3. Do we still need the `api.env('test')` ternary?

Reply:
Yeah, the tests won't run correctly if that's not there.

Reply:
Ah, right, that's why I did [this](https://github.com/WTW-IM/babel-preset-wtw-im/blob/133b96d5ab8c86cd673535df6b9497069df38a2a/default-settings.js#L7-L10) in the preset.

Reply:
I switched back to the wtw-im preset.